apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -61,7 +61,7 @@ export class WorkflowQueueService {
     await this.runJob.execute(
       RunJobCommand.create({
         jobId: job._id,
-        presend: job.presend,
+        presend: false,
```

Comment:
why we need`presend`?
or is it `present`?

Reply:
Presend is a legacy from when I implemented Digest and a feature that we did not finish with... Maybe we should remove it instead?

Reply:
> a feature that we did not finish with

If there's any plan to add this in the future^^, then maybe we can leave it as it is.
also perhaps rename it to something better to understand.

Reply:
I think there is right now only this flag exists so I think I will remove it...